_topic_maps/_topic_map.yml

```diff
@@ -4508,8 +4508,6 @@ Topics:
       File: virt-creating-vms-by-cloning-pvcs
   - Name: Connecting to VM consoles
     File: virt-accessing-vm-consoles
-  - Name: Specifying an instance type or preference
-    File: virt-specifying-instance-preference
```

Comment:
Is the file dropped?

Reply:
Some of the contents of this file are missing now? Like using instancetypes/preferences without inferencen and how to label data sources?

Reply:
@0xFelix 

> Is the file dropped?

No, it is now part of virt-creating-vms-from-instance-types.adoc. Now it is not a standalone section but belongs to Creating VMs - as you requested in Jira. 

> Some of the contents of this file are missing now? Like using instancetypes/preferences without inferencen and how to label data sources?

No, everything is in creating_vms_rh/virt-creating-vms-from-instance-types.adoc  - see https://85245--ocpdocs-pr.netlify.app/openshift-enterprise/latest/virt/virtual_machines/creating_vms_rh/virt-creating-vms-from-instance-types.html#creating-manifests-virtctl_virt-creating-vms-from-instance-types